pkg/middleware/stored_session.go

```diff
@@ -161,6 +162,11 @@ func (s *storedSessionLoader) refreshSession(rw http.ResponseWriter, req *http.R
 		logger.PrintAuthf(session.Email, req, logger.AuthError, "error saving session: %v", err)
 		return fmt.Errorf("error saving session: %v", err)
 	}
+	// Log if authenticated user details changed
```

Comment:
Turns out a shift in user/email during a refresh wasn't captured at all.

This covers our bases and aligns it to the `logger.PrintAuthf` calls that the main login flows us to log the authenticated user after sign-in.